docs/index.bs

```diff
@@ -364,6 +366,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
             1. Else, let it be initialized to a new {{Client}} object that represents the worker associated with |incumbentGlobal|.
             1. Let the {{ExtendableMessageEvent/ports}} attribute of |e| be initialized to |newPorts|.
             1. <a>Dispatch</a> |e| at |destination|.
+            1. Invoke [=Update Service Worker Extended Events Set=] with |serviceWorker| and |e|.
```

Comment:
Handle Functional Event runs https://github.com/w3c/ServiceWorker/pull/1065/files#diff-27b79860afe28f01aed4f1f6228367faR3082 after the callback steps given by other specs that are supposed to dispatch the functional events.